packages/graph/deployment/docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: "3.9"
+
 volumes:
   hash-graph-data:
-    external: true
```

Comment:
I don't have issues with this, but I'm curious why this change was made as part of this PR? Was it added accidentally previously, or has the thinking around it changed somehow?

Reply:
It's a QoL enhancement, so the caller does not need to create the volume themselves, it will be created (not destroyed) if it's not already existing.